Samples/MapScenarios/MapScenarios/Scenarios/WalkTime/WalkTimeViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  TimeTraveledViewController.swift
+//  MapScenarios
+//
+//  Created by Xiangwei Wang on 2/1/19.
+//  Copyright © 2019 Patrick Dunshee. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import PWCore
+import PWMapKit
+
+class WalkTimeViewController: TurnByTurnViewController {
+    
+    // GPS location manager - used to request location authentication
+    var clLocationManager: CLLocationManager!
+    
+    // Last update location
+    var lastUpdateLocation: PWUserLocation?
+    
+    // If the blue dot sit on the route path
+    var snappedLocation = false
```

Comment:
"If the blue dot sit on the route path" -> "If the blue dot is currently snapped to the route path"
snappedLocation -> snappingLocation
makes it clearer that that is the current state, not that the location was snapped once.